src/platform/utilities/cerner/index.js

```diff
@@ -6,10 +6,10 @@ export const isCernerLive = !environment.isProduction();
 
 export const getCernerURL = path => {
   const root = environment.isProduction()
-    ? 'https://patientportal.myhealth.va.gov'
-    : 'https://ehrm-va-test.patientportal.us.healtheintent.com';
+    ? 'https://patientportal.myhealth.va.gov/clear-session?to=https://patientportal.myhealth.va.gov'
+    : 'https://ehrm-va-test.patientportal.us.healtheintent.com/clear-session?to=https://ehrm-va-test.patientportal.us.healtheintent.com';
 
-  return `${root}${path}`;
+  return `${root}${encodeURIComponent(path)}`;
```

Comment:
This skips the `https://ehrm-va-test.patientportal.us.healtheintent.com` part in the `to` param.

Reply:
Doh

Reply:
Updated